### PR TITLE
Home mockup

### PR DESCRIPTION
**Pull request by pkakelas**
screenshot from 2014-05-10 18 02 12

### CODE REVIEW

**Comment by dionyziz** [comment]
Left-align Google button with username input

**Comment by dionyziz** [comment]
Right-align GitHub button with username input

**Comment by dionyziz** [comment]
Left- and right- align submit button with username input

**Comment by dionyziz** [comment]
Center Google and github text within buttons - exclude icon part from the width within which you align

File: mockups/home.html [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+                </li>
+                <li>
+                <a class="btn btn-block btn-social btn-github">
+                    <i class="fa fa-github"></i>Github
+                </a>
+                </li>
+                </ul>
+            </div>
+        </div>
+        <div id='last'>
```
Comment (vitsalis) [inline mockups/home.html]: last? I don't think that's a good id.

File: mockups/home.html [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+                        <td>
+                            <a href="user/view?username=vitsalis">vitsalis</a>
+                        </td>
+                        <td>Greece</td>
+                        <td>Coming soon</td>
+                    </tr> 
+                </tbody>
+            </table> 
+        </div>
+        <div id='screenshot'>
```
Comment (vitsalis) [inline mockups/home.html]: I don't think `screenshot` is a good id. You should better give it an id like `prevGame`

**Comment by pkakelas** [comment]
I find submit button more beautiful as it is. If you think its better the other way I can change it.

**Comment by pkakelas** [comment]
The leftmost pixel of google button has the same x with the username input. The rightmost pixel of github button has the same x with the username input. I don't understand what else I should do.

**Comment by dionyziz** [comment]
No, they don't.

ee5414d2-d859-11e3-98ee-0bbb81e40eb0

**Comment by dionyziz** [comment]
Why do you need the file mockups/google.png?

**Comment by pkakelas** [comment]
screenshot from 2014-05-13 17 32 32

**Comment by dionyziz** [comment]
Still need to center text within submit buttons.

**Comment by dionyziz** [comment]
Postponing this until we pair-program redesign this together w/ @pkakelas.

**Pull request closed without merging**